<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSystemLog</class>
 <widget class="QMainWindow" name="WidgetSystemLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Log</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resource.qrc">
    <normaloff>:/Resource/Network/HitMonitor.png</normaloff>:/Resource/Network/HitMonitor.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTextEdit" name="textEditSystemLog">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table border=&quot;0&quot; style=&quot;-qt-table-type: root; margin-top:4px; margin-bottom:4px; margin-left:4px; margin-right:4px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPauseLogDisplay"/>
   <addaction name="actionToggleTimestamp"/>
   <addaction name="actionClearBuffer"/>
  </widget>
  <action name="actionToggleTimestamp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/Time.png</normaloff>:/Resource/Generic/Time.png</iconset>
   </property>
   <property name="text">
    <string>Toggle Timestamp</string>
   </property>
  </action>
  <action name="actionClearBuffer">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/Clear.png</normaloff>:/Resource/Generic/Clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear Buffer</string>
   </property>
   <property name="toolTip">
    <string>Clear Buffer</string>
   </property>
  </action>
  <action name="actionPauseLogDisplay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Media/PauseMedia.png</normaloff>:/Resource/Media/PauseMedia.png</iconset>
   </property>
   <property name="text">
    <string>Pause Display</string>
   </property>
   <property name="toolTip">
    <string>Pause Display</string>
   </property>
  </action>
  <action name="actionShowWarnings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/Warning.png</normaloff>:/Resource/Generic/Warning.png</iconset>
   </property>
   <property name="text">
    <string>Show Warning Messages</string>
   </property>
   <property name="toolTip">
    <string>Show Warning Messages</string>
   </property>
  </action>
  <action name="actionShowInformation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/Properties.png</normaloff>:/Resource/Generic/Properties.png</iconset>
   </property>
   <property name="text">
    <string>Show Information Messages</string>
   </property>
  </action>
  <action name="actionShowSecurity">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Security/Security.png</normaloff>:/Resource/Security/Security.png</iconset>
   </property>
   <property name="text">
    <string>Show Security Messages</string>
   </property>
  </action>
  <action name="actionShowNotice">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/Notice.png</normaloff>:/Resource/Generic/Notice.png</iconset>
   </property>
   <property name="text">
    <string>Show Notice Messages</string>
   </property>
  </action>
  <action name="actionShowDebug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/Bug.png</normaloff>:/Resource/Generic/Bug.png</iconset>
   </property>
   <property name="text">
    <string>Show Debug Messages</string>
   </property>
  </action>
  <action name="actionShowError">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/Exit.png</normaloff>:/Resource/Generic/Exit.png</iconset>
   </property>
   <property name="text">
    <string>Show Error Messages</string>
   </property>
  </action>
  <action name="actionShowCritical">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/Critical.png</normaloff>:/Resource/Generic/Critical.png</iconset>
   </property>
   <property name="text">
    <string>Show Critical Messages</string>
   </property>
   <property name="toolTip">
    <string>Show Critical Messages</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Generic/Copy.png</normaloff>:/Resource/Generic/Copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>textEditSystemLog</sender>
   <signal>copyAvailable(bool)</signal>
   <receiver>actionCopy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
